cmake/MacOSXBundleInfo.plist.in

```diff
@@ -44,5 +44,28 @@
     <string>${MACOSX_SPARKLE_UPDATE_PUBLIC_KEY}</string>
     <key>SUFeedURL</key>
     <string>${MACOSX_SPARKLE_UPDATE_FEED_URL}</string>
+    <key>CFBundleDocumentTypes</key>
+    <array>
+        <dict>
+            <key>CFBundleTypeExtensions</key>
+			<array>
```

Comment:
Redo the indents in this file with spaces instead of tabs, please.

Reply:
Fixed!